### DISCUSSION TITLE
Where can we get a dedicated Stripe Payment Link for our bill that is due

### DISCUSSION THREAD

**Discussion by sakettawde** [vercel/community#1791] [Help] [answered]
### Summary

How can we get a dedicated Stripe Payment Link for our bill that is due? As our Credit Card is rejected due to the need for additional security inputs.

### Additional information

This is the same link that is present in the email sent by Vercel after the account is suspended due to non-payment. We have this issue with all International Indian credit cards due to RBI regulations.

This is the error we currently see
![Screenshot 2023-03-15 at 6 01](https://user-images.githubusercontent.com/14161564/225311016-d26485f2-cb33-414a-9c94-48873e67412c.png)

**Comment by amyegan** [comment, marked as answer]
I'm sorry the link isn't working, @sakettawde. Please [open a new `Billing` case](https://vercel.com/help#:~:text=Submit%20a%20case%20to%20our%20Customer%20Success%20Team.) or send an email to [billing@vercel.com](mailto:billing@vercel.com) and we will help you solve this problem while keeping your information private and secure 🙏
